app/javascript/components/enrollment/steps/AdditionalPlannedTravel.js

```diff
@@ -11,7 +11,7 @@ import { stateOptions } from '../../../data/stateOptions';
 class AdditionalPlannedTravel extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { ...this.props, current: { ...this.props.currentState }, modified: {}, errors: {} };
+    this.state = { ...this.props, current: { ...this.props.currentState }, modified: {}, errors: {}, isEditMode: window.location.href.includes('edit') };
```

Comment:
can you just pass in the `edit_mode` prop in enrollment and use that here?

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/441588e6694ae61c4bd1da5b7da3be4de8fd5aa9